pkg/operator/ceph/object/rgw.go

```diff
@@ -14,39 +14,32 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package rgw for the Ceph object store.
+// Package object for the Ceph object store.
 package object
 
 import (
 	"fmt"
 
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/rook/rook/pkg/clusterd"
-	"github.com/rook/rook/pkg/daemon/ceph/client"
-	rgwdaemon "github.com/rook/rook/pkg/daemon/ceph/rgw"
+	daemonconfig "github.com/rook/rook/pkg/daemon/ceph/config"
+	cephconfig "github.com/rook/rook/pkg/operator/ceph/config" // TODO: rename to just config
```

Comment:
is this TODO still planned? Looks like it's not possible since you have two "config" packages